MiniApp/Classes/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -92,3 +113,15 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
         }
     }
 }
+
+class LocationManager: NSObject, CLLocationManagerDelegate {
```

Comment:
Why are you implementing `CLLocationManagerDelegate` if you don't use the delegate methods ?

Reply:
My bad. Removed it